packages/flutter_tools/templates/module/android/Flutter.tmpl/src/main/java/io/flutter/facade/Flutter.java

```diff
@@ -27,72 +27,94 @@ private Flutter() {
     // to prevent instantiation
   }
 
-  public static void startInitialization(Context applicationContext) {
+  /**
+   * Initiates the Dart VM. Calling this method at an early point may help decreasing time to first
+   * frame for a subsequently created {@link FlutterView}.
+   *
+   * @param applicationContext the application's {@link Context}
+   */
+  public static void startInitialization(@NonNull Context applicationContext) {
     FlutterMain.startInitialization(applicationContext, null);
   }
 
-  public static Fragment createFragment(String route) {
+  /**
+   * Creates a {@link FlutterFragment} managing a {@link FlutterView}.
+   *
+   * @param route the route String for the {@link FlutterView}
+   * @return a {@link FlutterFragment}
+   */
+  @NonNull
+  public static FlutterFragment createFragment(String route) {
     final FlutterFragment fragment = new FlutterFragment();
     final Bundle args = new Bundle();
     args.putString(FlutterFragment.ARG_ROUTE, route);
     fragment.setArguments(args);
     return fragment;
   }
 
-  public static View createView(final Activity activity, final Lifecycle lifecycle, final String route) {
+  /**
+   * Creates a {@link FlutterView} linked to the specified {@link Activity} and {@link Lifecycle}
+   * and whose contents is determined using the specified route {@link String}.
```

Comment:
Perhaps rephrase to indicate that we are sending the route to the dart code that can dispatch on it using defaultRouteName.